exporters/prometheus/exporter.go

```diff
@@ -180,7 +184,9 @@ func (c *collector) Collect(ch chan<- prometheus.Metric) {
 		if !c.disableScopeInfo {
 			scopeInfo, err := c.scopeInfo(scopeMetrics.Scope)
 			if err != nil {
-				otel.Handle(err)
+				if err != errScopeInvalid {
```

Comment:
I think that this code would be more idiomatic:

```go
if err == errScopeInvalid {
	continue
}
if err != nil {
	otel.Handle(err)
	continue
}
```


Reference: https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

Reply:
Even looking again at that doc, I'm not sure that bit is more idiomatic. And it seems harder to read, since there are 2 conditions handling the same error.
But that's a nit, so I've made the change.